common/djangoapps/student/admin.py

```diff
@@ -3,6 +3,8 @@
 from django.contrib.auth import get_user_model
 from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
 from django.utils.translation import ugettext_lazy as _
+from django.conf import settings
```

Comment:
It was needed in a previous commit, but not anymore. Strange that the CI didn't warn against it.